lib/Unexpected.js

```diff
@@ -488,6 +488,8 @@ Unexpected.prototype.fail = function (arg) {
             var value = arg[key];
             if (key === 'diff') {
                 error.createDiff = value;
+            } else if (key === 'stack') {
+                error._stack = value;
```

Comment:
We should consider naming this property something else like `originalError` and store the store a reference to the error instance.